Library/Formula/folly.rb

```diff
@@ -0,0 +1,46 @@
+class Folly < Formula
+  desc "An open-source C++ library developed and used at Facebook."
```

Comment:
Could you make it a little less ambiguous? Maybe even just `grab bag of awesome` etc.